/**:
  ros__parameters:
    odom_freq: 20
    transform_tolerance: 0.2
    global_frame_id: "map"
    odom_frame_id: "odom"
    base_frame_id: "base_link
    scan1_frame_id: "laser_right"
    scan2_frame_id: "laser_left"
    scan1_topic: "/scan_1"
    scan2_topic: "/scan_2"

    footprint_frame_id: "base_footprint"
    
    laser_min_range: 0.0
    laser_max_range: 100000000.0
    scan_increment: 1
    initial_pose_x: 0.0
    initial_pose_y: 0.0
    initial_pose_a: 0.0
    num_particles: 1200
    alpha_threshold: 0.5
    expansion_radius_position: 0.1
    expansion_radius_orientation: 0.2
    extraction_rate: 0.1
    range_threshold: 0.3
    sensor_reset: false
    # odom_fw_dev_per_fw: 0.19
    # odom_fw_dev_per_rot: 0.0001
    # odom_rot_dev_per_fw: 0.013
    # odom_rot_dev_per_rot: 0.005
    odom_fw_dev_per_fw: 0.19
    odom_fw_dev_per_rot: 0.001
    odom_rot_dev_per_fw: 0.013
    odom_rot_dev_per_rot: 0.5
    laser_likelihood_max_dist: 0.5
